client/x/evmstaking/keeper/keeper.go

```diff
@@ -144,7 +149,7 @@ func (k Keeper) ProcessStakingEvents(ctx context.Context, height uint64, logs []
 				continue
 			}
 			ev.StakeAmount.Div(ev.StakeAmount, gwei)
-			if err = k.ProcessCreateValidator(ctx, ev); err != nil {
+			if err = k.HandleCreateValidatorEvent(ctx, ev); err != nil {
```

Comment:
Can we combine all these events to a single enqueue since all of them share the same msg type

Reply:
Yes, all types of events are converted to `types.QueuedMessage` and enqueue to a single queue. Did you mean to combine all `Handle*Event` to a single func?

Reply:
I don't think we can combine all `Handle*Event` function to a single function. Because each event has different fields. If we want to make a single function, we need to move the logics for handling events (converting each event to `QueuedMessage` and enqueuing) out to `ProcessStakingEvents`. If I misunderstand your comment, please correct me.